.github/actions/container-build_test_release/action.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright 2023 NWChemEx-Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: container action to build ParallelZone
+
+description: container action from the base image to build ParallelZone
+
+inputs:
+  base_tag:
+    description: 'tag of the base image'
+    required: false
+    default: 'stable'
+  token:
+    type: string
+    required: true
+  user:
+    type: string
+    required: true
+  cmake_version:
+    type: string
+    required: false
+    default: ''
+  gcc_version:
+    type: string
+    required: false
+    default: ''
+  clang_version:
+    type: string
+    required: false
+    default: ''
```

Comment:
Don't worry about it in this PR, but by time you get to the NWChemEx repo you're going to have a ton of parameters for versions. I think YAML supports maps. So you probably could get by with a map from dependency name to version number (and using the default version, as specified by NWXCMake, for any unspecified dependencies).